frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -278,6 +319,20 @@ class Agenda extends React.Component {
           icon='calendar'
           breadcrumbsList={state.breadcrumbsList}
         />
+        <div className='agendaPage__updateMsg'>
```

Comment:
this should be in a frontend_lib component since we will need it in different place